models/transforms.py

```diff
@@ -91,7 +91,10 @@ def __call__(self, vid):
             size = None
         else:
             size = self.size
-        return torch.nn.functional.interpolate(vid, size=size, scale_factor=scale, mode=interpolation, align_corners=False)
```

Comment:
This should not throw a warning when `self.size` is `int` and keep previous behavior when it is a `tuple`.